flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/sink/DataSinkWriterOperatorFactory.java

```diff
@@ -36,17 +36,31 @@ public class DataSinkWriterOperatorFactory<CommT>
                 YieldingOperatorFactory<CommittableMessage<CommT>> {
 
     private final Sink<Event> sink;
+    private final boolean isBatchMode;
     private final OperatorID schemaOperatorID;
 
-    public DataSinkWriterOperatorFactory(Sink<Event> sink, OperatorID schemaOperatorID) {
+    public DataSinkWriterOperatorFactory(
+            Sink<Event> sink, boolean isBatchMode, OperatorID schemaOperatorID) {
         this.sink = sink;
+        this.isBatchMode = isBatchMode;
         this.schemaOperatorID = schemaOperatorID;
     }
 
     @SuppressWarnings("unchecked")
     @Override
     public <T extends StreamOperator<CommittableMessage<CommT>>> T createStreamOperator(
             StreamOperatorParameters<CommittableMessage<CommT>> parameters) {
+
+        if (isBatchMode) {
```

Comment:
just a personal flavor,I think use `isBounded` in code is more aligned with flink's stream concept(unbounded stream and bounded stream )